test/multicluster/minikube_base_multicluster_test.go

```diff
@@ -134,4 +140,73 @@ func TestKubernetesBasicMultiCluster(t *testing.T) {
 			verifyNuoSQL(t, context.Namespace, admin0, "demo")
 		})
 	})
+
+	testlib.RunOnNuoDBVersionCondition(t, ">=5.0.2", func(version *semver.Version) {
```

Comment:
I'll update the version to `5.0.3` in case the feature doesn't make it for NuoDB 5.0.2.